### PR TITLE
Test aad-pod-id chart on sandbox

### PR DESCRIPTION
**Pull request by hannah38**
Add in aad-pod-id chart to sandbox
Remove kustomize sandbox for aad-pod-id to test chart
Fixes for sandbox 1.6.2 version included

**Does this PR introduce a breaking change?** (check one with "x")

Only changing on sandbox as this version needs tested
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/admin/pod-identity/patches/sandbox/nmi-daemonset.yaml [hunk 1]
```diff
@@ -14,3 +14,12 @@ spec:
       containers:
         - name: nmi
           image: "mcr.microsoft.com/k8s/aad-pod-identity/nmi:1.6.2"
+          args:
```
Comment (adusumillipraveen) [inline k8s/namespaces/admin/pod-identity/patches/sandbox/nmi-daemonset.yaml:17]: do u need these patches ? I thought u removed them from kustomize

File: k8s/sandbox/common/admin/aad-pod-identity.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: helm.fluxcd.io/v1
```
Comment (adusumillipraveen) [inline k8s/sandbox/common/admin/aad-pod-identity.yaml:2]: Also, why not add it as a different Kustomize name aad-pod-identity?

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  name: aad-pod-identity
+  namespace: admin
+spec:
+  releaseName: aad-pod-identity
+  chart:
+    repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
+    name: aad-pod-identity
+    version: 1.6.2
+  values:
+    installMICException: false
```
Comment (timja) [inline k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml:16]: shouldn't this be true?
```suggestion
    installMICException: true
```

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    forceNameSpaced: true
+    nmi:
+      probePort: "8080"
```
Comment (timja) [inline k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml:19]: do we need this?

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml (hunk 3 above)
Comment (hannah38) [reply k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml:16]: Fine to be corrected - but this is what that configures - https://github.com/Azure/aad-pod-identity/blob/v1.6.2/charts/aad-pod-identity/templates/mic-exception.yaml - which I couldn't find in our current setup

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml (hunk 4 above)
Comment (hannah38) [reply k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml:19]: Can change it (8085 default), included it as 8080 is what we have set currently

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml (hunk 3 above)
Comment (timja) [reply k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml:16]: https://github.com/Azure/aad-pod-identity#1-deploy-aad-pod-identity

> # For AKS clusters, deploy the MIC and AKS add-on exception by running -
kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/mic-exception.yaml

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml (hunk 4 above)
Comment (timja) [reply k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml]: we don't rely on that I would use the default

File: k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml (hunk 3 above)
Comment (hannah38) [reply k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml:16]: Cool thanks, I've removed it as it's true by default

**Pull request merged**